src/main/java/net/fabricmc/accesswidener/AccessWidenerClassVisitor.java

```diff
@@ -43,9 +50,15 @@ public void visit(int version, int access, String name, String signature, String
 		className = name;
 		classAccess = access;
 
+		AccessWidener.Access classAccess = accessWidener.getClassAccess(name);
+
+		if (strictRecords && "java/lang/Record".equals(superName) && AccessWidener.ClassAccess.isExtendable(classAccess)) {
+			throw new UnsupportedOperationException(String.format("Cannot modify record (%s) access to be extendable", name));
```

Comment:
Should we fail explicitly or simply ignore these, like what we are doing with interface static fields? non-final interface static fields cause classfile format errors when loaded by jvm, but record violations apparently won't hurt the jvm class loading.

Reply:
I think ingore now 👍Crashing like this now would really need a major version bump. With v3 we can possibly have something this strict.

Loom's validator should really fail on both of this situations (and possibly more): https://github.com/FabricMC/fabric-loom/blob/dev/0.12/src/main/java/net/fabricmc/loom/task/ValidateAccessWidenerTask.java#L97

I tottaly forgot about this PR.